docs/documentation/getting-started/millionaires.md

```diff
@@ -51,15 +84,24 @@ java -jar cs.jar amphora get-secrets
 The output should resemble the following:
 
 !!! note
-    The output you see will differ wrt. identifiers and the `creation-date` tag.
+    The output you see will differ wrt. identifiers and the `creation-date` tag. <br>
+    Nevertheless, it will output both secrets, uploaded by Elon and Jeff even though
+    we are authenticated as Elon. This is to the fact of authentication, but not
```

Comment:
"This is **due** to the fact **that** authentication, but not authorization, is currently built into Carbyne Stack."

Reply:
rephrazed to match the actual behaviour:
> Nevertheless, it will output both secrets, uploaded by Elon and Jeff even
though we are authenticated as Elon. However, accessing secrets to which a 
user is not authorized - as defined by the policy attached to the secret -
will result in an `Unauthorized` error.